addons/mrp/models/stock_move.py

```diff
@@ -100,10 +100,13 @@ class StockMove(models.Model):
 
     def _unreserve_initial_demand(self, new_move):
```

Comment:
is it ok if id doesn't unreserve "in place" anymore?
actually i don't even get the usecase.. do we have a test somewhere?

Reply:
test_link_assign_9 in test_move.py

Reply:
i'm pretty this test isn't related since _unreserve_initial_demand doesn't do anything in stock and test_move si in stock

+ when completely commenting _unreserve_initial_demand the tests are green T_T